### ISSUE TITLE
Data Analysis Team tasks for Week of 10/24

### ISSUE THREAD

**Issue by kdahlquist** [kdahlquist/GRNmap#266]
The tasks for this week:
- @kdahlquist has "review requested" for 
  - [ ] #230 (degree distribution charts)
  - [ ] #245, generating and vetting input workbooks (after requested changes are made
- @bklein7 and @Nwilli31 are working on the following:
  - [x] #245, generating and vetting input workbooks, finishing off requested changes
  - [x] #241, update 16_tests workbooks on beta branch to be consistent with our formatting guidelines 
  - [x] #265 start the model runs using the v1.4.4 source
- @khorstmann and @maggie-oneil are working on the following:
  - [ ] #259  @maggie-oneil and @khorstmann will work on getting the SBET toolbox up and running, starting with betweenness centrality.

Note that nobody seems to be working on #170, so I changed the priority to 0.5 on that one.

**Comment by maggie-oneil** [comment]
Worked with @khorstmann to create a guide for the functions of the SBEToolbox, specifically the view functions, along with random networks "small world" and "ring-lattice." To find powerpoint guide, please see @khorstmann 's comment for the document.

**Comment by khorstmann** [comment]
Maggie and I ran through virtually every program on SBEToolbox with the 4-node sample network we talked about yesterday. We weren't able to get to the 21-gene network as it took us much longer to get through it than expected. On Monday we can scroll through everything and decide which things to run on our larger test to see how those work. We also need to add on more  information on the downloading, installation, etc of the program so that we can maybe use the powerpoint for documentation of future usage of the program. 

Powerpoint can't be uploaded here because it is too large, but here's the link to my lab notebook, where it is uploaded under the Oct 25 page http://www.openwetware.org/wiki/Kristen_M._Horstmann_Lab_Notebook

**Comment by kdahlquist** [comment]
Sorry, I forgot to stop by after lab yesterday because I was in a hurry to get home.  I've looked over the PowerPoint--it looks like there will be a lot to talk about next week.

**Comment by kdahlquist** [comment]
I've booked the first floor LSB conference room (118) for next Monday's (10/31) lab meeting so that we can look at the PowerPoints on the big screen.  See you all there at 3:00.

**Comment by khorstmann** [comment]
http://www.openwetware.org/wiki/Kristen_M._Horstmann_Lab_Notebook#October_25.2C_2016 
@bengfitzpatrick here is the link to my Lab Notebook. Under the entry for Oct 25, there is the powerpoint we showed today

**Comment by khorstmann** [comment]
Searched through https://github.com/kdahlquist/DahlquistLab/tree/master/data/GRNmap_input_workbooks where we were directed to find the networks to test SBE Toolbox on, but couldn't find the weighted networks. So far had just been working on the unweighted networks when we realized.
Could @bklein7 and @Nwilli31 please let us know where their weighted networks are? Thanks!

**Comment by khorstmann** [comment]
@maggie-oneil an through all the stats and the adjacency for the 3 unweighted networks. POwerpoint can be found in the Nov 1 entry here: http://www.openwetware.org/wiki/Kristen_M._Horstmann_Lab_Notebook#November_1.2C_2016

Determined that the stats (or, at least shortest path, and therefore, betweenness centrality) assume the networks are undirected, which is a bit of an issue for our networks. I'm sure we will discuss in the meeting on monday how to continue

**Comment by Nwilli31** [comment]
I looked at the wt output sheet parameters again and realized that I recorded the incorrect numbers for the Penalty term and the LSE. I had them switched, so I changed the numbers on the powerpoint. I reuploaded the file on my openwetware page and on the Dahlquist Repository.

**Comment by kdahlquist** [comment]
The Data Analysis team meetings will take place in LSB 118 for the rest of the semester.

**Comment by bklein7** [comment]
All 5 output sheets from the round of model runs done for issue #265 have been uploaded to the [Dahlquist Lab Repository](https://github.com/kdahlquist/DahlquistLab/tree/master/data/GRNmap_input_workbooks), so @khorstmann and @maggie-oneil can access them there.

**Comment by maggie-oneil** [comment]
@khorstmann and I input two weighted networks to see if the SBEToolbox took the weights into account when running statistics of the graphs. After running both it appears as though the toolbox does not take into account the weights or direction of edges. We then looked into both yEd and gephi to see if those graphing tools would be able to take direction and weight into account, but found both to be lacking in a cursory review of the programs. The powerpoint for this week is in PDF format and linked directly to this comment. 
[Weighted SBEToolbox.pdf](https://github.com/kdahlquist/GRNmap/files/582501/Weighted.SBEToolbox.pdf)

**Comment by kdahlquist** [comment]
It looks like there are solutions out there for finding the shortest path in a weighted, directed network, but it might be harder to find "off the shelf" software.

A Google search, turned up this:  http://www.geeksforgeeks.org/shortest-path-exactly-k-edges-directed-weighted-graph/

**Comment by maggie-oneil** [comment]
@khorstmann and I looked into the graph applications and stats applications of the program Gephi and it appears to be more useful and user-friendly than SBEToolbox. We used both a simple, unweighted network and the weighted output sheet for GLN3 in our analysis of the program. The powerpoint of our findings is attached directly to this comment.
[Giphe.pptx](https://github.com/kdahlquist/GRNmap/files/593585/Giphe.pptx)
